src/tools/miri/src/lib.rs

```diff
@@ -17,7 +17,7 @@
 #![feature(derive_coerce_pointee)]
 #![feature(arbitrary_self_types)]
 #![feature(iter_advance_by)]
-#![cfg_attr(not(bootstrap), feature(duration_from_nanos_u128))]
+#![feature(duration_from_nanos_u128)]
```

Comment:
FWIW there is a [corresponding `cfg_select!` on `bootstrap`](https://github.com/rust-lang/miri/blob/38a5e6b571ac8f501af5c4b201f194c99e1d8b78/src/clock.rs#L50) in the code that matches this... I guess the pattern you are grepping for involves `cfg.*bootstrap` and so the `cfg_select!` did not show up?